src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,133 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int threshold;
```

Comment:
can be calculated from table and LOAD_FACTOR at any time, why have it as a field?